appengine-cron-emulator/build.gradle

```diff
@@ -13,7 +13,8 @@ repositories {
 
 dependencies {
     compile group: 'com.squareup.okhttp', name: 'okhttp', version: '2.7.5'
-    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-xml', version: '2.9.9'
+    compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: '2.9.9'
```

Comment:
Is there any way for these to use the variables like `JACKSON_VERSION` in `build.gradle`?

Reply:
Good call. I refactored them to use a variable here.

I'm not sure if you were asking if I could reference the values we already have in `api/build.gradle`; I'll probably keep them separate for now as I don't know if there's much benefit and it would create a circular dep (at least, logically).

Reply:
Didn't realize the dependency.  This works, thanks.